clients/admin-ui/src/features/datastore-connections/system_portal_config/forms/ConnectorParameters.tsx

```diff
@@ -40,6 +42,33 @@ import {
 import { ConnectionConfigFormValues } from "../types";
 import ConnectorParametersForm from "./ConnectorParametersForm";
 
+type TestDataProps = {
+  succeeded?: boolean;
+  timestamp: string;
+};
+
+const TestData: React.FC<TestDataProps> = ({ succeeded, timestamp }) => {
```

Comment:
It looks like ConnectionGrid has pretty much this same component. Can we pull this component out into its own file, and use the same component in both places to keep things dry? It could probably live as a file under `datastore-connections`

Reply:
Good call.  Done!